### PR TITLE
Fix 475: Incorrect content in client screen, if data are not available

### PR DESCRIPTION
**Pull request by mayank-kgp**
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by mayank-kgp** [comment]
@therajanmaurya This is the screen on error.

incclient

**Comment by mayank-kgp** [comment]
@therajanmaurya Please review this and suggest me the required changes.

**Comment by therajanmaurya** [comment]
ImageView and TextView should be at the center horizontal and vertical center.

**Comment by mayank-kgp** [comment]
@therajanmaurya I have updated. :)

incclient

**Comment by therajanmaurya** [comment]
@mayank-kgp Please fix this PR

**Comment by mayank-kgp** [comment]
@therajanmaurya fixed

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientdetails/ClientDetailsFragment.java [hunk 1]
```diff
@@ -670,7 +683,18 @@ public void onItemClick(AdapterView<?> adapterView, View view, int i,
 
     @Override
     public void showFetchingError(String s) {
-        Toast.makeText(getActivity(), s, Toast.LENGTH_SHORT).show();
+        if (Network.getConnectivityStatusString(getContext()).equals("Not connected to Internet")) {
```
Comment (therajanmaurya) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientdetails/ClientDetailsFragment.java]: Use ```Network.isOnline(Context)``` and please do not use any hardcoded string.

**Comment by mayank-kgp** [comment]
@therajanmaurya I have made changes

**Pull request closed without merging**